src/Exceptions/MissedRemoteDocumentationUrlException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace RonasIT\Support\AutoDoc\Exceptions;
+
+use Exception;
+
+class MissedRemoteDocumentationUrlException extends Exception
+{
+    public function __construct($message = null, $code = 0, Exception $previous = null)
+    {
+        $message = $message ?? 'Remote documentation url missed in config';
```

Comment:
let's add the information about the env variable name which should be filled to fix it